src/main/java/org/opensearch/security/dlic/rest/api/ActionGroupsApiAction.java

```diff
@@ -73,8 +73,8 @@ public ActionGroupsApiAction(final Settings settings, final Path configPath, fin
 	}
 
 	@Override
-	public List<Route> routes() {
-		return routes;
+        public List<ReplacedRoute> replacedRoutes() {
```

Comment:
check with tab size set to 2:
https://github.com/afazel/security/blob/use_replaced_routes_in_rest_api/src/main/java/org/opensearch/security/dlic/rest/api/ActionGroupsApiAction.java?ts=2
or 4:
https://github.com/afazel/security/blob/use_replaced_routes_in_rest_api/src/main/java/org/opensearch/security/dlic/rest/api/ActionGroupsApiAction.java?ts=4